vnext/local-cli/runWindows/utils/autolink.js

```diff
@@ -6,95 +6,359 @@
 // @ts-check
 
 const execSync = require('child_process').execSync;
-const path = require('path');
 const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+
+const {newSpinner} = require('./commandWithProgress');
```

Comment:
If we're using a spinner or trying to be interactive we should try to avoid the synchronous fs calls we added and await fa.promises, etc instead. We can otherwise get something that stutters.

Looks like some code is already async?

Reply:
I can change this in a future iteration.